ui/app/config.js

```diff
@@ -99,6 +110,19 @@ ConfigScreen.prototype.render = function () {
   )
 }
 
+function rpcValidation (newRpc, state) {
+  if (validUrl.isWebUri(newRpc)) {
+    state.dispatch(actions.setRpcTarget(newRpc))
+  } else {
+    var appendedRpc = `http://${newRpc}`
+    if (validUrl.isWebUri(appendedRpc)) {
+      state.dispatch(actions.displayWarning('URIs require the appropriate HTTP/HTTPS prefix.'))
```

Comment:
I think it's ok to warn them that it's definitely a prefix problem, but it's less ok to auto-fill a prefix, since we may have misled them that we fixed it.  Not 100% on that, though...